ScaleformUI_Lua/src/Menus/PauseMenus/LobbyMenu/MainView.lua

```diff
@@ -562,38 +562,63 @@ function MainView:ProcessControl()
         return
     end
 
-    if (IsControlPressed(2, 172)) then
+    if (IsDisabledControlJustPressed(2, 172)) then
```

Comment:
why "just" pressed? the ControlPressed is done to allow the "keep button pressed to scroll"

Reply:
This line check only client started press arrow up.
And keep repeat client holding arrow up on line 570